src/briefcase/console.py

```diff
@@ -167,7 +170,7 @@ def wait_bar(
             the message must already be escaped; defaults False.
         """
         wait_bar = Progress(
-            TextColumn("     "),
+            TextColumn("    "),
             BarColumn(bar_width=20, style="black", pulse_style="white"),
             TextColumn(message),
             transient=True,
```

Comment:
I noticed the pulsing bar of the Wait Bar is actually shifted one character over from where the Progress Bar starts. This aligns them. It is most apparent when installing `java` for instance.